src/raftstore/store/worker/compact.rs

```diff
@@ -15,16 +15,17 @@ use raftstore::store::keys;
 use raftstore::store::engine::Iterable;
 use util::worker::Runnable;
 use util::rocksdb;
-use storage::{CF_RAFT, CF_LOCK};
+use storage::CF_RAFT;
 
 use rocksdb::{DB, WriteBatch, Writable};
 use std::sync::Arc;
 use std::fmt::{self, Formatter, Display};
 use std::error;
 
 pub enum Task {
-    CompactLockCF {
+    CompactRangeForCF {
         engine: Arc<DB>,
```

Comment:
I still think we don't need to pass engine here, we can create the Runner with the engine and then use it.